free-programming-books.md

```diff
@@ -763,7 +755,6 @@ Original Source: [List of freely available programming books](http://web.archive
 
 ### C
 * [A Tutorial on Pointers and Arrays in C](http://home.netcom.com/~tjensen/ptr/pointers.htm)
-* [Advanced Linux Programming](http://www.advancedlinuxprogramming.com/)
```

Comment:
Good, I just found out that this was not free: "All rights reserved. No part of this book may be reproduced [...] without written permission from the publisher"